### PR TITLE
0.16: release notes for submariner PR#3021

### PR DESCRIPTION
**Pull request by aswinsuryan**
Release notes for submariner-io/submariner#3021

### CODE REVIEW

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.16.5
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md]:
```suggestion
## v0.16.6
```

File: src/content/community/releases/_index.en.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ weight = 40
...
+
+* Fixed an issue in Health Checker where a restart of health checker during event like a leader reelection would lead to invalid Pinger
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Health check counters on the Gateway resource now report correct information after a gateway leader re-election occurs.
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md (hunk 1 above)
Comment (aswinsuryan) [reply src/content/community/releases/_index.en.md]: Done.

File: src/content/community/releases/_index.en.md (hunk 2 above)
Comment (aswinsuryan) [reply src/content/community/releases/_index.en.md]: Done

**Review by tpantelis — APPROVED** [review]

**Pull request merged**